packages/typespec-test/test/openai/generated/typespec-ts/.tshy/esm.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
are those files under .tshy folder auto generated when you run build with tshy ? and they are not supposed to be modified ? I am just curious as I don't see any code is generating those file?

Reply:
`tshy` auto generates these files and are safe to check in.